operator/test/manifests/test/tekton-chains/simple-copy-pipeline.yaml

```diff
@@ -26,44 +26,48 @@ spec:
           - name: IMAGE_DST
-            description: Reference of the image buildah will push.
-          - name: BUILDER_IMAGE
-            description: The location of the buildah builder image.
-            default: quay.io/buildah/stable:v1.18.0
-        workspaces:
-          - name: sslcertdir
-            optional: true
+            description: Reference of the image skopeo will push.
+          - name: SKOPEO_IMAGE
+            description: The location of the skopeo image.
+            default: quay.io/skopeo/stable:v1.9.0
+          - name: srcTLSverify
+            description: Verify the TLS on the src registry endpoint
+            type: string
+            default: "true"
+          - name: destTLSverify
+            description: Verify the TLS on the dest registry endpoint
+            type: string
+            default: "true"
         results:
           - name: IMAGE_DIGEST
             description: Digest of the image just built.
           - name: IMAGE_URL
             description: Reference of the image the pipeline will produce.
         steps:
           - name: copy
-            image: $(params.BUILDER_IMAGE)
+            env:
+              - name: HOME
+                value: /tekton/home
+            image: $(params.SKOPEO_IMAGE)
             script: |
-              [[ "$(workspaces.sslcertdir.bound)" == "true" ]] && CERT_DIR_FLAG="--cert-dir $(workspaces.sslcertdir.path)"
-              echo "# Pulling image from: $(params.IMAGE_SRC)"
-              buildah \
-                ${CERT_DIR_FLAG} \
-                pull \
-                $(params.IMAGE_SRC)
-              echo "# Pushing image to: $(params.IMAGE_DST)"
-              buildah \
-                ${CERT_DIR_FLAG} \
-                push \
-                --digestfile /tmp/image-digest \
-                $(params.IMAGE_SRC) docker://$(params.IMAGE_DST)
-              cat "/tmp/image-digest" | tee $(results.IMAGE_DIGEST.path)
-              echo "$(params.IMAGE_DST)" | tee $(results.IMAGE_URL.path)
-            volumeMounts:
-              - name: varlibcontainers
-                mountPath: /var/lib/containers
-        volumes:
-          - name: varlibcontainers
-            emptyDir: {}
+              set -o errexit
+              set -o pipefail
+              if [ "$(params.IMAGE_SRC)" != "" ] && [ "$(params.IMAGE_DST)" != "" ] ; then
+                skopeo copy \
+                  docker://"$(params.IMAGE_SRC)" docker://"$(params.IMAGE_DST)" \
+                   --digestfile /tmp/image-digest \
+                   --src-tls-verify="$(params.srcTLSverify)" \
+                   --dest-tls-verify="$(params.destTLSverify)"
+                echo "$(params.IMAGE_DST)" > "$(results.IMAGE_URL.path)"
+                cat "/tmp/image-digest" > "$(results.IMAGE_DIGEST.path)"
+              else
+                return 1
+              fi
+            securityContext:
+              runAsNonRoot: true
+              runAsUser: 65532
```

Comment:
Is this required ?

Reply:
`runAsUser` is not required. Although, defining SecurityContext is a good practice. Do you think it should be removed for any reason from this pipeline manifest?

Reply:
> defining SecurityContext is a good practice.

runAsNonRoot should be enough I think. There should be a reason to run as a specific user otherwise I don't see the need for that.

Reply:
That makes sense. I'll quickly make the changes.

Reply:
Actually I found that this was required :) 
More about it here: https://github.com/openshift-pipelines/pipeline-service/pull/423